src/Microsoft.Health.Blob/Configs/BlobDataStoreRequestOptions.cs

```diff
@@ -13,6 +13,8 @@ public class BlobDataStoreRequestOptions
 
         public int ServerTimeoutInMinutes { get; set; } = 2;
 
-        public int DownloadMaximumConcurrency { get; set; } = 1;
+        public int DownloadMaximumConcurrency { get; set; } = 5;
```

Comment:
default value is 5 in SDK, here https://github.com/Azure/azure-sdk-for-net/blob/71d3750b48129b42b30d1a95ed9a906aba72648b/sdk/storage/Azure.Storage.Common/src/Shared/Constants.cs#L198